src/Utilities/Resources/xlf/Strings.de.xlf

```diff
@@ -185,7 +185,7 @@
       </trans-unit>
       <trans-unit id="Tracking_SourceWillBeCompiledDependencyWasModifiedAt">
         <source>{0} will be compiled as {1} was modified at {2}.</source>
-        <target state="translated">{0} wird als {1} kompiliert, wurde geändert bei {2}.</target>
```

Comment:
(And `Tracking_SourceWillBeCompiledMissingDependency` will be wrong as well).
The current translation assumes that "as" is meant as identity.. ("like a ...", "into ...") and not "because"